src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -28,8 +35,8 @@ <h2 class="card-pf-title">
     </p>
   </div>
 </div>
-<div ng-if="$ctrl.isMiniLayout" class="card-pf card-pf-aggregate-status card-pf-aggregate-status-mini" ng-class="{'card-pf-accented': $ctrl.shouldShowTopBorder}">
-  <h2 class="card-pf-title">
+<div ng-if="$ctrl.isMiniLayout" class="card-pf card-pf-aggregate-status card-pf-aggregate-status-mini" ng-class="{'card-pf-accented': $ctrl.shouldShowTopBorder}" ng-style="$ctrl.spinnerHeight">
```

Comment:
Not sure why the ng-style was added here?  It seemed to work in the `show-spinner` div below.